### ISSUE TITLE
 Post Excerpt of Post Element is not working!

### ISSUE THREAD

**Issue by sshobhi776** [ThemeFuse/Brizy#825]
Hi,

I have added Post Element on my page with Post #Post Title, #Post Excerpt but only Post's title is showing on the front end, no text for #Post Excerpt.

Please see the attached images for the details.

![post backend](https://user-images.githubusercontent.com/41860320/47959550-208b5900-e00d-11e8-93d3-5829e57d4768.png)

![post frontend](https://user-images.githubusercontent.com/41860320/47959612-7ca2ad00-e00e-11e8-9746-d782b8f6fbfb.png)

Thank you.

**Comment by sshobhi776** [comment]
Template style/layout for Post is also not working.

Thank you.

**Comment by elecstorm** [comment]
I have the same issue.
Excerpts only appear in posts object when I explicitly write them in a blog post Excerpt field.
I checked different themes, different Settings/Reading options: excerpts not working in posts object.

**Comment by peleroberts** [comment]
The team are aware, just waiting for a fix so we can have full post dynamic control

**Comment by elecstorm** [comment]
Hi! I checked it yesterday - still doesn't work. #Post Content does not exist in posts object also. But it is in brizy templates.

**Comment by bagrinsergiu** [comment]
Hi all,

Sorry for the delay. On my local install Free - 1.0.45 + Pro 0.0.12 it is working like you can see in this screenshot: https://jumpshare.com/v/x1eTJHhxNpLScxxglY8n  

We added some fixes for DC in last pro update.

Thanks for reporting.
Sergiu

**Comment by dmccan** [comment]
I have Brizy 1.0.45 and Brizy Pro 0.0.12 installed. The excerpt does not show.

![brizy-post-excerpt-1](https://user-images.githubusercontent.com/2622805/48266780-1b0b8580-e3f6-11e8-9861-5cf736e65526.png)

![brizy-post-excerpt-2](https://user-images.githubusercontent.com/2622805/48266791-1f37a300-e3f6-11e8-8c7a-77bd9a0a1b16.png)

**Comment by sshobhi776** [comment]
Hi bagrinsergiu,

As you see the dmccan feedback on latest update and issue, the issue is still not fixed.

I have checked that if you manually insert the Post's Excerpt (using Yoast plugin) then it shows there on front end.

We supposed to display the post's excerpt from first paragraph of post's content.

Please check.

Thank you

**Comment by dmccan** [comment]
I gathered the issues related to archive templates in #819.

**Comment by dmccan** [comment]
I tried this with 1.0.47 and 0.0.13 and it still does not work.

**Comment by bagrinsergiu** [comment]
Hi all,

Sorry for the delay. I made a lot of tests with Dynamic Content.  

Homepage template test video (Dynamic Content + WP Fields): https://jumpshare.com/v/QOL3GRAd2mlMDVKYaMTX

---

Homepage template test video (Dynamic Content + The Content): https://jmp.sh/xbnszzF

---

Homepage template test video (Dynamic Content + ACF Fields): https://jmp.sh/3erMrCp

- [x] Post Excerpt is working 
- [x] Pagination is working   
- [x] The_Content + Brizy Typography Style ( Color styles is planned in next update 0.0.14)
- [x] ACF Fileds

**Comment by bagrinsergiu** [comment]
### Category Template 

---

Category template tebrst video (Dynamic Content + WP Fields): https://jmp.sh/aQEtmao

---

Category template test video (Dynamic Content + The Content): https://jmp.sh/iGZULSj

---

Homepage template test video (Dynamic Content + ACF Fields): https://jmp.sh/JztpuHB

---   

- [x] Post Excerpt is working
- [x] Pagination is working
- [x] The_Content + Brizy Typography Style ( Color styles is planned in next update 0.0.14)
- [ ] ACF Fileds

**Comment by bagrinsergiu** [comment]
### Search template 

---

Search template test video (Dynamic Content + WP Fields): https://jmp.sh/TSBWp16

---

Search template test video (Dynamic Content + The Content): https://jmp.sh/z2GE1vU

--- 

Search template test video (Dynamic Content + ACF Fields): https://jmp.sh/ObyNyE0

---

[ok] Post Excerpt is working
[ok] Pagination is working
[ok] The_Content + Brizy Typography Style ( Color styles is planned in next update 0.0.14)
[ok] ACF Fileds

**Comment by bagrinsergiu** [comment]
Hi all,

My tests environnement is: 
Brizy Free: 1.0.52
Brizy Pro: 0.0.14

----

Post Excerpt + Pagination Demo link: https://jmp.sh/QSeTrTg
The Content Colors Demo link: https://jmp.sh/lYGIEe3

Sorry for the delay and thanks for reporting. 
Can you confirm if it is working for you?
Sergiu

**Comment by peleroberts** [comment]
Sergiu let me know if you want access to my personal site so you can see for yourself if anything 👍

**Comment by elecstorm** [comment]
bagrinsergiu, I already checked it in 14 - doesn't work. Try to go to the pages, and do the same in any page. It doesn't work there. #post content also doesn't work (in templates it works).

**Comment by peleroberts** [comment]
Wow, #Post_Content (color) does not work for me in templates, tried with Astra and latest wp 2019 theme.

**Comment by sshobhi776** [comment]
Here is my understanding is that-  #Post_Excerpt for a post is not generated automatically hence template is also not showing this. As I mentioned earilier, If I add custom Excerpt in post then It display there.

But It should work like - #Post_Excerpt should fetch few sentances from first paragraph as other Themes do.

I hope this helps.

Thanks!

**Comment by CarlosA8** [comment]
yeah, i am having the same issues... just got the pro and been trying to find a work around. :(

**Comment by leokoo** [comment]
Hmm, this doesn't work for me too :(
